data/35e/wizards_of_the_coast/core/dungeon_masters_guide/dmg_equip.lst

```diff
@@ -322,11 +323,11 @@ Rhino Hide				PROFICIENCY:ARMOR|Hide		TYPE:Magic.Armor.Medium.ArmorProfMedium.Ar
 Darkwood Buckler		PROFICIENCY:SHIELD|Buckler		TYPE:Shield.Light.ShieldProfLight.Specific.Nonmetal				COST:5	WT:5		SLOTS:0	ACCHECK:-1								EQMOD:DARK												SOURCEPAGE:p.221	SPELLFAILURE:5										BONUS:COMBAT|AC|1|TYPE=Shield																	SPROP:This nonmagical light wooden shield is made out of darkwood and often favored by rogues and wizards. It has no enhancement bonus, but its construction material makes it lighter than a normal wooden shield. It weighs 2-1/2 pounds and has no armor check penalty. No aura (nonmagical);
 Darkwood Shield		PROFICIENCY:SHIELD|Shield (Heavy)	TYPE:Shield.Heavy.ShieldProfHeavy.Specific.Nonmetal				COST:7	WT:10				ACCHECK:-2		CRITMULT:x2	CRITRANGE:1	DAMAGE:1d4	EQMOD:DARK							WIELD:OneHanded	SIZE:M	SOURCEPAGE:p.221	SPELLFAILURE:15										BONUS:COMBAT|AC|2|TYPE=Shield																	SPROP:This nonmagical heavy wooden shield is made out of darkwood. It has no enhancement bonus, but its construction material makes it lighter than a normal wooden shield. It weighs 5 pounds and has no armor check penalty. No aura (nonmagical)
 Lion's Shield		PROFICIENCY:SHIELD|Shield (Heavy)	TYPE:Magic.Enhancement.Shield.Heavy.ShieldProfHeavy.Specific		COST:20	WT:15				ACCHECK:-2		CRITMULT:x2	CRITRANGE:1	DAMAGE:1d4	EQMOD:MAGIC_COST|5000.Masterwork (Armor).PLUS2S.STEEL		WIELD:OneHanded	SIZE:M	SOURCEPAGE:p.221	SPELLFAILURE:15										BONUS:COMBAT|AC|2|TYPE=Shield																	SPROP:This +2 heavy steel shield is fashioned to appear to be a roaring lion's head. Three times per day as a free action, the lion's head can be commanded to attack (independently of the shield wearer), biting with the wielder's base attack bonus (including multiple attacks, if the wielder has them) and dealing 2d6 points of damage. This attack is in addition to any actions performed by the wielder. Moderate conjuration; CL 10th
-Mithral Heavy Shield	PROFICIENCY:SHIELD|Shield (Heavy)	TYPE:Shield.Heavy.ShieldProfHeavy.Specific					COST:20	WT:15				ACCHECK:-2		CRITMULT:x2	CRITRANGE:1	DAMAGE:1d4	EQMOD:MITHRAL_SHLD					WIELD:OneHanded	SIZE:M	SOURCEPAGE:p.221	SPELLFAILURE:15										BONUS:COMBAT|AC|2|TYPE=Shield																	SPROP:This heavy shield is made of mithral and thus is much lighter than a standard steel shield. It has a 5%% arcane spell failure chance and no armor check penalty. It weighs 5 pounds. No aura (nonmagical)
+Mithral Heavy Shield	PROFICIENCY:SHIELD|Shield (Heavy)	TYPE:Shield.Heavy.ShieldProfHeavy.Specific					COST:20	WT:15				ACCHECK:-2		CRITMULT:x2	CRITRANGE:1	DAMAGE:1d4	EQMOD:MITHRAL_SHLD					WIELD:OneHanded	SIZE:M	SOURCEPAGE:p.221	SPELLFAILURE:15										BONUS:COMBAT|AC|2|TYPE=Shield																	SPROP:This heavy shield is made of mithral and thus is much lighter than a standard steel shield. It has a 5% arcane spell failure chance and no armor check penalty. It weighs 5 pounds. No aura (nonmagical)
```

Comment:
Removing the escape % will cause an error.

Reply:
as above, no error is logged. SPROP is not working as documented.